edx/edx-app-ios
# Basic Reviews Prompting

### Description

[MA-3107](https://openedx.atlassian.net/browse/MA-3107)
[MA-3137](https://openedx.atlassian.net/browse/MA-3137)
[MA-3147](https://openedx.atlassian.net/browse/MA-3147)

This PR intends to incorporate the app review feature that will be triggered at the end of the video experience. 

### How to test this PR

1. Login into the app
2. Navigate to a course
3. Play a video and wait for it to finish
4. The app review popup will appear at the end of the video
